modules/virt-creating-vm-cli.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-cli_{context}"]
+= Using the CLI to create a virtual machine
+
+You can create a virtual machine from a `virtualMachine` manifest.
+
+.Procedure
+
+. Edit the `VirtualMachine` manifest, according to the following example for a RHEL virtual machine, and save it as `vm.yaml`:
```

Comment:
I think we should make it even more explicit that this is an example and should not be used verbatim. We also don't need to dictate the filename; I would nix that part.

Suggestion (using attributes for RHEL's short and long names):

```
. Edit the `VirtualMachine` manifest for your VM. For example, the following manifest configures a {op-system-base-full} virtual machine:
+
.Example manifest for a {op-system-base} VM
[source,yaml]
```

You can also make this collapsible; IIRC, the syntax is:

```
. Example manifest for a {op-system-base} VM
[%collapsible]
====
[source,yaml]
----
<YAML block>
---
====
```

Reply:
I was following the "majority". :-) I looked for other examples in CNV docs and copied them. 
I like the collapsible tag. Will try that.